src/trigrams.py

```diff
@@ -1 +1,57 @@
 """This module contains an implementation of a trigram algorithm."""
+import io
+import re
+import random
+import sys
+
+
+def pulltext(file_path):
+    """Open a text file and turn it into a list of single word strings."""
+    text_file = io.open(file_path).read()
+    return re.findall('\w+', text_file)
```

Comment:
Nice job diving into regex, I know you have seen regex before from your 301 project!  Consider not stripping out punctuation so that the text is realistic.